src/_guides/language/sound-faq.md

```diff
@@ -264,7 +264,11 @@ The following image shows examples of type inference in strong mode clean code:
 <a name="is-it-optional"></a>
 ### I liked Dart before. Can I still write Dart code the way I always did?
 
-Yes, because strong mode is optional, you can write Dart code as before.
+Strong mode is optional in Dart 1.x, so you can write code as before.
```

Comment:
This could be a bit clearer. Also, shouldn't we link to the new page? Maybe:

Yes, but we recommend migrating your code now. For help on getting ready for Dart 2.0, see [/dart-2.0](Dart 2.0 Updates).

_(Yeah, I broke my rule about not using page names in links, but I couldn't think of a better way to refer to the page.)_